tests/run_tests.sh

```diff
@@ -59,7 +59,7 @@ else
   cmd="tests/runner.py --test-path ${path} ${sdk_dir} --test-pattern $substring --coverage-file ${coverage_file}"
 fi
 
-(cd ${BASE_DIR}; PYTHONPATH=./:./data_steward:${PYTHONPATH} python ${cmd})
+(cd "${BASE_DIR}"; PYTHONPATH=./:./data_steward:${PYTHONPATH} python ${cmd})
```

Comment:
Do you mind adding quotes around `${PYTHONPATH}` and `${cmd}` as well?
```suggestion
(cd "${BASE_DIR}"; PYTHONPATH=./:./data_steward:"${PYTHONPATH}" python "${cmd}")
```

Reply:
This doesn't work because `cmd` is already a string, sorry about that